cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -1161,6 +1167,9 @@ func newServer(conf rootConfig) (*Server, error) {
 
 	logger := sglog.Scoped("server", "periodically reindexes enabled repositories on sourcegraph")
 
+	q := Queue{}
+	q.Init(conf.backoffDuration, conf.maxBackoffDuration, logger)
```

Comment:
`Init` is mainly so we pass logger a single time. Alternatively we could just pass logger to queue methods and scope in each method. I'd still need to pass the two config values to queue on Server initialization, otherwise they'd need to be members of Server which they should not need to be